zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2763,95 @@ def check_lvmlockd_log():
+        if pv_name not in  pv_dev_names_by_vg_name:
+            if pv_name not in pv_dev_names_by_vg_name:
+                pv_dev_names_by_vg_name[pv_name] = []
+            pv_dev_names_by_vg_name[pv_name].append(vg_name)
+    return pv_dev_names_by_vg_name
+
+
+def get_multipath_info():
+    output = bash.bash_o("multipath -ll").strip().splitlines()
+    wwid_to_mpath = {}  # WWID -> 多路径设备名
+    dev_to_mpath = {}  # 底层设备名 -> 多路径设备名
+    mpath_to_wwid = {}  # 多路径设备名 -> WWID（新增反向映射）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)